src/e2e/src/utils/assert.utils.ts

```diff
@@ -1,64 +1,57 @@
-import { ElementFinder, ExpectedConditions as EC, ProtractorBrowser } from 'protractor'
+import { ProtractorBrowser } from 'protractor'
 import { MainPage } from '../pages/main.page'
 import { getBrowser } from '../browser-manager'
-import { waitForElementToBeClickable, waitForElementToBeVisible } from './browser.utils'
-import { wait } from './async.utils'
+import { waitForElementToBeVisible } from './browser.utils'
 
 let mainPage: MainPage
 let browser: ProtractorBrowser
 
 export async function confirmationMessageAsserts(currencies: string, transaction: string, expectedResult: string, notional: string, timeout: boolean) {
   browser = await getBrowser()
   mainPage = new MainPage(browser)
-  let tradeWord: string
-  let parsedTradeText: string
+  const tradeWord: string = transaction === 'buy' ? 'bought' : 'sold';
```

Comment:
This method was a tad *racey*. Reworked it a little but kept the overall assertions. This could be better thought out.